swap/tests/testutils/mod.rs

```diff
@@ -226,11 +226,6 @@ impl TestContext {
             self.bob_starting_balances.btc - self.btc_amount - lock_tx_bitcoin_fee
         );
 
-        // unload the generated wallet by opening the original wallet
-        self.bob_monero_wallet.re_open().await.unwrap();
```

Comment:
You cannot remove this! You need this on Bob's side unless you reload his default wallet.

Alice: We re-load the default wallet (because concurrent swaps...)
Bob: We **don't** reload the default wallet - we just sweep the wallet that was `generated_from_keys` - in the tests we send the money to the original wallet (in reality it would go to any user wallet specified with address upon startup).